presto/testing/common/fixtures.py

```diff
@@ -15,11 +15,18 @@
 import pytest
 
 from . import test_utils
-
+from ..integration_tests.common_fixtures import get_scale_factor
```

Comment:
`common` should not be depending on `integration_tests`. The `get_scale_factor` fixture should probably be moved to `common`.